linalg/ode.cpp

```diff
@@ -568,7 +638,6 @@ const real_t AM3Solver::a[] =
 const real_t AM4Solver::a[] =
 {251.0/720.0,646.0/720.0,-264.0/720.0, 106.0/720.0, -19.0/720.0};
 
-
```

Comment:
Two empty line between class implementations is typical styling we use -- please do not make this type of changes.

Reply:
I have tried to correct this one. However, things have changed quite a bit so this is also caused by github not really being able to link things up.

Thank you for the 2-line style comment -- I learned something :)